src/RoomsSDKAdapter.js

```diff
@@ -134,4 +144,119 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
 
     return this.getRoomObservables[ID];
   }
+
+  /**
+   * Invoke request to listActivities on adapter
+   *
+   * @param {object} Params to send to conversation api method
+   * @returns
+   * @private
+   */
+  fetchActivities({roomId, earliestActivityDate}) {
+    const {limit} = this;
+    const conversationId = deconstructHydraId(roomId).id;
+
+    logger.debug('ROOM', 'fetchActivities()', ['called with', {conversationId, limit, earliestActivityDate}]);
+
+    let maxDate = earliestActivityDate;
```

Comment:
Not sure if there is value in doing this

Reply:
Since you wanted the name changed to `earliestActivityDate` It made sending deconstructed variables to the underlying odd.